synapse/handlers/federation.py

```diff
@@ -1443,13 +1454,16 @@ async def get_room_complexity(
 
     async def _sync_partial_state_room(
         self,
-        destination: str,
+        destination: Optional[str],
+        destinations: Collection[str],
```

Comment:
can we come up with better names for these (`initial_destination` and `other_destinations`)? or just combine them into one and arrange for `destination` to be first?

Reply:
I'm going to go with `initial_destination` and `other_destinations`.

I'm hesitant to combine them just yet, since they've got different properties: `initial_destination` is likely to be up, while `other_destinations` is a mess